### DISCUSSION TITLE
Loading stylesheets in ActionMailer previews

### DISCUSSION THREAD

**Discussion by dimitrisnl** [ElMassimo/vite_ruby#436] [Q&A]
Hello!

I'm migrating from Webpacker to Vite, and so far everything has been smooth. The only issue that I face, is that I don't see any styles applied when I visit the action mailer previews `http://localhost:5000/rails/mailers/...`

Using `vite_image_tag` for the images works fine, but the stylesheets don't work. I'm unsure how to work around this issue. Any ideas?

**Comment by ElMassimo** [comment]
Hi Dimitrios!

Please provide more information about **how** the _stylesheets don't work_.

Have you added `vite_stylesheet_tag` to your mailer layout? Are the stylesheet tags being rendered, do the stylesheets 404, are you running the Vite dev server?

**Reply by dimitrisnl** [reply]
@ElMassimo Your workaround here https://github.com/ElMassimo/vite_ruby/pull/93 solved the issue for me!
